### ISSUE TITLE
For training, each question should have exactly 1 answer

### ISSUE THREAD

**Issue by SandeepBhutani** [google-research/bert#392]
I can see Squad 2.0 related files are there. 
When I run training for Squad 2.0 I get following error:

```
Traceback (most recent call last):
  File "run_squad.py", line 1283, in <module>
    tf.app.run()
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/platform/app.py", line 125, in run
    _sys.exit(main(argv))
  File "run_squad.py", line 1159, in main
    input_file=FLAGS.train_file, is_training=True)
  File "run_squad.py", line 268, in read_squad_examples
    "For training, each question should have exactly 1 answer.")
ValueError: For training, each question should have exactly 1 answer.
```

**Comment by StalVars** [comment]
Try with argument "--version_2_with_negative True"

**Comment by JeevaTM** [comment]
I tried

> Try with argument "--version_2_with_negative True"

I tried this argument. I still have the issue. 

```
python run_squad.py --vocab_file=$BERT_LARGE_DIR/vocab.txt --bert_config_file=$BERT_LARGE_DIR/bert_config.json --init_checkpoint=$BERT_LAR
GE_DIR/bert_model.ckpt --do_train=True --train_file=$SQUAD_DIR/train-v2.0.json --do_predict=True --predict_file=$SQUAD_DIR/dev-v2.0.json --train_batch_size=24 --learning_rate=
3e-5 --num_train_epochs=2.0 --max_seq_length=384 --doc_stride=128 --output_dir=gs://uniquetestbucket/neural-output --use_tpu=True \ --tpu_name=$TPU_NAME \ --version_2_with_neg
ative True
WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.
Traceback (most recent call last):
  File "run_squad.py", line 1283, in <module>
    tf.app.run()
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/platform/app.py", line 125, in run
    _sys.exit(main(argv))
  File "run_squad.py", line 1159, in main
    input_file=FLAGS.train_file, is_training=True)
  File "run_squad.py", line 268, in read_squad_examples
    "For training, each question should have exactly 1 answer.")
ValueError: For training, each question should have exactly 1 answer.
```

and

```
python run_squad.py --vocab_file=$BERT_LARGE_DIR/vocab.txt --bert_config_file=$BERT_LARGE_DIR/bert_config.json --init_checkpoint=$BERT_LAR
GE_DIR/bert_model.ckpt --do_train=True --train_file=$SQUAD_DIR/train-v2.0.json --do_predict=True --predict_file=$SQUAD_DIR/dev-v2.0.json --train_batch_size=24 --learning_rate=
3e-5 --num_train_epochs=2.0 --max_seq_length=384 --doc_stride=128 --output_dir=gs://uniquetestbucket/neural-output --use_tpu=True \ --tpu_name=$TPU_NAME \ --version_2_with_neg
ative=True
WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.
Traceback (most recent call last):
  File "run_squad.py", line 1283, in <module>
    tf.app.run()
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/platform/app.py", line 125, in run
    _sys.exit(main(argv))
  File "run_squad.py", line 1159, in main
    input_file=FLAGS.train_file, is_training=True)
  File "run_squad.py", line 268, in read_squad_examples
    "For training, each question should have exactly 1 answer.")
ValueError: For training, each question should have exactly 1 answer.
```
